backend/src/v1/data/models/models.ts

```diff
@@ -1,15 +1,13 @@
-export type Translation = {
-    id: string,
-    sourceWord: Word,
-    targetWord: Word;
-};
-
 export type Word = {
```

Comment:
These models are left over from when i was trying to follow a [clean architecture](https://blog.cleancoder.com/uncle-bob/2012/08/13/the-clean-architecture.html) approach so these were going to be "domain" models. I've gotten burned in Android/Java/Kotlin land when letting types exposed/generated by data layer permeate the app (made trying to migrate to a new lib a huge headache). But I suppose with TS's typing that's probably less of an issue since it's based on structure and not the actual definitions of the types defined in third-party packages.